src/main/java/frc/robot/subsystems/Leds.java

```diff
@@ -0,0 +1,84 @@
+package frc.robot.subsystems;
+
+import com.spikes2212.command.DashboardedSubsystem;
+import com.spikes2212.dashboard.Namespace;
+import edu.wpi.first.wpilibj.AddressableLED;
+import edu.wpi.first.wpilibj.AddressableLEDBuffer;
+import frc.robot.RobotMap;
+
+import java.util.function.Supplier;
+
+public class Leds extends DashboardedSubsystem {
```

Comment:
This is not a subsystem, therefore it shouldn't inherit `DashboardedSubsystem`. It is a service (so it needs to be in the services package as well), and if needed, just create here a RootNamespace object.